libs/shared/contentful/src/lib/factories.ts

```diff
@@ -23,6 +29,65 @@ import {
 } from './queries/services-with-capabilities';
 import { ContentfulErrorResponse } from './types';
 
+export const CapabilityServiceByPlanIdsQueryFactory = (
```

Comment:
I see this file getting a bit out of hand which is totally not on you, but we should definitely be putting each query's factories in the query directory alongside all of the other materials.

Reply:
As per my other comment, moved to its own file as per our Slack discussion. Thank you!